addons/mail/static/src/web/thread_service_patch.js

```diff
@@ -199,6 +199,13 @@ patch(ThreadService.prototype, "mail/web", {
         }
         delete this.store.followers[follower.id];
     },
+
+    async updateAvatar(threadId, data) {
```

Comment:
Should be moved outside of `web` to work in public page (see reported crashed above)